src/it/it-abstract-versions-report-001/pom.xml

```diff
@@ -0,0 +1,66 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>localhost</groupId>
+  <artifactId>it-abstract-versions-report-001</artifactId>
+  <version>1.0</version>
+  <packaging>pom</packaging>
+  <name>abstract-versions-report</name>
+  <url>http://localhost/</url>
+
+  <description>
+    Testing command line invocation of dependency- and plugin-updates-report
+  </description>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>localhost</groupId>
+        <artifactId>dummy-api</artifactId>
+        <version>1.1</version>
+      </dependency>
+      <dependency>
+        <groupId>localhost</groupId>
+        <artifactId>dummy-impl</artifactId>
+        <version>1.1</version>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+  <dependencies>
+    <dependency>
+      <groupId>localhost</groupId>
+      <artifactId>dummy-api</artifactId>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <artifactId>maven-clean-plugin</artifactId>
+          <version>2.2</version>
```

Comment:
Please use the latest versions .... in parent pom we have defined properties like: `maven-clean-plugin.version`
so we can use as: 
```
 <version>@maven-clean-plugin.version@ </version>
```

Reply:
Actually, I wanted to demonstrate that the dependency/plugin update reports work, so we'd actually want to use some older versions, but I think it's better to use mocked artifacts (from the mock repository) instead, like `dummy-maven-plugin`.